README.md

```diff
@@ -21,18 +21,11 @@ The namespace must already exist in FireCloud.
 
 ## The Bag
 
-Currently the lambda expects one TSV file in the bag, and then splits it up into two TSV files, uploading each one in
-order to FireCloud. The logic for splitting up the TSV assumes the format of the way Boardwalk currently exports
-data, and does not belong in the lambda. Work is in progress to change this.
-
-The lambda should expect a [BagIt ](https://en.wikipedia.org/wiki/BagIt ), with its data directory containing two TSV
-files, one with the participants and one with the samples. We need to finalize how the lambda will know which file is
-participants and which one is samples.
+Currently the lambda expects a [BagIt](https://en.wikipedia.org/wiki/BagIt) containing two two TSV files, `participant.tsv` and `sample.tsv`, which it  uploads in to FireCloud in that order. We need to finalize how the lambda will know which file is participants and which one is samples.
```

Comment:
1. `two two`
2. You can remove the sentence "We need to finalize...", as the previous sentence does have it finalized.